commune_app/all_models/Chore.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from .users import User
+
+
+class Chore(models.Model):
+    title = models.CharField(max_length=50)
+    date = models.DateTimeField(auto_now_add=True)
+    budget = models.DecimalField(max_digits=7, decimal_places=2, null=True)
+    assign = models.ForeignKey(User, on_delete=models.CASCADE)
+    description = models.TextField(max_length=300)
+
+    def __str__(self):
+        return self.title
+
+    def snippet(self):
```

Comment:
maybe it will be more accurate like:
```suggestion
    def description_snippet(self):
```